test/unit/offset.js

```diff
@@ -462,9 +462,9 @@ testIframe("offset/body", "body", function( $ ) {
 test("chaining", function() {
 	expect(3);
 	var coords = { "top":  1, "left":  1 };
-	equal( jQuery("#absolute-1").offset(coords).selector, "#absolute-1", "offset(coords) returns jQuery object" );
-	equal( jQuery("#non-existent").offset(coords).selector, "#non-existent", "offset(coords) with empty jQuery set returns jQuery object" );
-	equal( jQuery("#absolute-1").offset(undefined).selector, "#absolute-1", "offset(undefined) returns jQuery object (#5571)" );
+	equal( jQuery("#absolute-1").offset(coords).jquery, jQuery.fn.jquery, "offset(coords) returns jQuery object" );
```

Comment:
> It's killing two birds with one stone

Yeah! Or "a bird in the hand is worth two in the bush"

This property indeed serve for two complitly different things, but i don't think that's good, i think these should be exposed explicitly then otherwise. Besides, i'm not sure that was an original intent.

> the presence of the property identifies an object as a jQuery

This could always be done by other means, like: 

```js
function test() {}
new test().constructor.name // test

// i.e. we could do

function jQuery() {}
jQuery.prototype.constructor = jQuery;
$().constructor.name // "jQuery"
```
Or other countless ways to do it. Not saying we should do something about it, it might be too late, like with `$.each vs $.map` callback arguments case :-(